operator/performance_tests/test.sh

```diff
@@ -0,0 +1,103 @@
+kubectl config set-context --current --namespace=load-test
+export CLIENT_ID="$(kubectl get secret oauthclient -o jsonpath='{.data.client_id}' | base64 --decode)"
+export CLIENT_SECRET="$(kubectl get secret oauthclient -o jsonpath='{.data.client_secret}' | base64 --decode)"
+export ENCODED_CREDENTIALS=$(echo -n "$CLIENT_ID:$CLIENT_SECRET" | base64)
+export TOKEN=$(curl -s -X POST "https://oauth2.$DOMAIN/oauth2/token" -H "Authorization: Basic $ENCODED_CREDENTIALS" -F "grant_type=client_credentials" -F "scope=read" | jq -r '.access_token')
+export K6POD="$(kubectl get pods --selector=app=k6 -o jsonpath='{.items[0].metadata.name}')"
+
+cat > load-testing/values.yaml <<EOF
```

Comment:
We don't need this as long as we can give domain value in the helm install command